### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by Saravana-Gunaseelan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+The file browser needs to know what type of media to show to the user.
+This can be provided in the onCreate,
+
+```java
+@Override
+protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+    setMediaObjectFilters(new MediaObjectFilter[]{MediaObjectFilter.LIGHT_FIELD_IMAGE});
```
Comment (PuneetKohli) [inline README.md]: Missing `}`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+public class MainActivity extends BaseActivity {}
+```
+
+### 2. Implement the abstract methods
+
+```java
+@Override
+public Uri getMediaUriToEdit() {
+    // the media uri to be sent to the edit intent
+    return uri;
```
Comment (PuneetKohli) [inline README.md]: Can we add a section on where the `uri` field is set?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+@Override
+public void fileBrowserMediaUri(Uri uri) {
+    // uri from the file browser
```
Comment (PuneetKohli) [inline README.md]: Does this code need to set `uri`?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+@Override
+public void editMediaUri(Uri uri) {
+    // uri from edit intent
```
Comment (PuneetKohli) [inline README.md]: Same here, does this `uri` need to be passed to the `BaseActivity.uri` variable?

File: README.md (hunk 2 above)
Comment (Saravana-Gunaseelan) [reply README.md]: it depends on the app, the uri should always be set to the content that needs editing when the user clicks the edit icon

File: README.md (hunk 4 above)
Comment (Saravana-Gunaseelan) [reply README.md]: no, this is the uri for the apps once the user returns from the edit intent, it does not need to be sent anywhere. The app needs to choose what to do with it.

Same for the above, but it comes from file browser.

**Pull request merged**